ensembl/datacheck/app.py

```diff
@@ -246,31 +247,36 @@ def job_list():
             job['status'] = 'failed'
 
     # if request.is_json:
-    return jsonify(jobs)
-    # else:
-    # Need to pass some data to the template...
-    # return render_template('ensembl/datacheck/list.html')
+    if format=='json': 
```

Comment:
I wonder if this job_list can't be returned by a query made in command-line tools, hence request would be in JSON. @james-monkeyshines any advice about a potential CLI tool interacting with this route?
So the test would be a little more complicated (see the previous commented line)

Reply:
@james-monkeyshines ? any advice? is this route called by a cli specifying only request headers ? `request.is_json`

Reply:
The client should set the header, instead of adding the format (imho)